library/src/scala/tasty/Reflection.scala

```diff
@@ -2493,21 +2493,24 @@ trait Reflection { reflection =>
   // IMPLICIT SEARCH //
   /////////////////////
 
-  // TODO: this should not be top level
-  /** Find an implicit of type `T` in the current scope given by `ctx`.
-   *  Return an `ImplicitSearchResult`.
-   *
-   *  @param tpe type of the implicit parameter
-   *  @param ctx current context
-   */
-  def searchImplicit(tpe: Type): ImplicitSearchResult
+  val Implicits: ImplicitsModule
+
+  trait ImplicitsModule { self: Implicits.type =>
+    /** Find a given instance of type `T` in the current scope.
+     *  Return an `ImplicitSearch`.
+     *
+     *  @param tpe type of the implicit parameter
+     */
+    def search(tpe: Type): ImplicitSearch
+  }
 
-  type ImplicitSearchResult <: AnyRef
+  /** Result of a given instance search */
+  type ImplicitSearch <: AnyRef
```

Comment:
I find the previous name is better for readability. Meanwhile, maybe use "implicit instance search" in the docs for consistency and precision: as Scala2 implicits are not given instances.

Reply:
I will go back to ImplicitSearchResult now that we don't have to use it in source